cmd/utils/flags.go

```diff
@@ -755,9 +755,9 @@ var (
 		Name:  "proxy.proxied",
 		Usage: "Specifies whether this validator will be proxied by a proxy node",
 	}
-	ProxyEnodeURLPairFlag = cli.StringFlag{
-		Name:  "proxy.proxyenodeurlpair",
-		Usage: "proxy enode URL pair separated by a semicolon.  The format should be \"<internal facing enode URL>;<external facing enode URL>\"",
+	ProxyEnodeURLPairsFlag = cli.StringFlag{
```

Comment:
Consider using `StringSliceFlag` instead. I believe that will enable using the list type for this variable in the `toml` representation as well. https://godoc.org/github.com/urfave/cli#StringSliceFlag